examples/ts/create-order-position-with-takeprofit-stoploss.ts

```diff
@@ -45,13 +45,13 @@ async function example () {
     const params = {
         'stopLoss': {
             'triggerPrice': stop_loss_trigger_price,
-            // set a 'price' to act as limit order or leave commented for a market order
-            // 'price': stop_loss_trigger_price * 0.98,
+            // 'triggerPriceType': 'mark', // mark, last, index, bid, ask
```

Comment:
@ttodua the thing is since this is not unified yet it should be in the outer level  of `params` (same level as stopLoss, takeProfit, etc) because `stopLoss/takeProfit` are going to be omitted

Reply:
ok, I'll draft this PR for now